### ISSUE TITLE
Can I set the pre-trained model to depth_anything_v2_metric_vkitti_vitl.pth for further fine-tuning on outdoor datasets?

### ISSUE THREAD

**Issue by booker-max** [DepthAnything/Depth-Anything-V2#214]
Hello, I would like to fine-tune the metric depth network on my own outdoor dataset. Can I use depth_anything_v2_metric_vkitti_vitl.pth as the pre-trained model to start training instead of depth_anything_v2_vitl.pth? Would this approach yield better results?

**Comment by LiheYoung** [comment]
I think you can try both strategies. But personally, I would recommend using `depth_anything_v2_vitl.pth` as a pre-training source, since it is trained on much larger-scale data and is more general.

**Comment by eirtech** [comment]
> Hello, I would like to fine-tune the metric depth network on my own outdoor dataset. Can I use depth_anything_v2_metric_vkitti_vitl.pth as the pre-trained model to start training instead of depth_anything_v2_vitl.pth? Would this approach yield better results?


Hi, 
Did you already try both of them and maybe compare which one had a better result?
